src/middlewares/index.js

```diff
@@ -0,0 +1,19 @@
+const {
+  userAuthentication,
+  GroupAuthentication
+} = require('./authentication/');
+
+const middleware1 = {
```

Comment:
`middleware1` is the best name, really? 😫

Reply:
done